src/sort.jl

```diff
@@ -0,0 +1,71 @@
+import Base.@_inline_meta
+import Base.Order: Ordering, Forward, ReverseOrdering, ord
+import Base.Sort: Algorithm, defalg, lt, sort
```

Comment:
```suggestion
import Base.Sort: Algorithm, lt, sort
```

I suggest to not import `Base.Sort.defalg`. This probably yields a strange method error when user try `sort!`ing `MVector`, saying that `BitonicSortAlg` is not defined for `sort!`.

Reply:
I removed the import. Though `defalg` for `StaticVector` tests for immutability, so that should have been fine.